src/workspacer/ConfigContext.cs

```diff
@@ -5,6 +5,7 @@
 using System.IO;
 using System.Linq;
 using System.Windows.Forms;
+using workspacer.Shared.Layout;
```

Comment:
I noticed you used workspacer as the primary namespace. My environment was configured to use namespaces based on folder hierarchy. I can revert this to workspacer or move the remaining layouts to `Shared.Layout`